### ISSUE TITLE
Dockerfile MAINTAINER command

### ISSUE THREAD

**Issue by pablo-de-andres** [simphony/osp-core#717]
The `MAINTAINER` command is deprecated, and `LABEL maintainer="name"` should be used instead

**Comment by kysrpex** [comment]
Actually, docker [suggests](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated):

```dockerfile
LABEL org.opencontainers.image.authors="SvenDowideit@home.org.au"
```

**Comment by pablo-de-andres** [comment]
Yes, it looks like they have changed the key. This will have to be updated in wrapper_development and some wrappers as well.
